tests/manager/api/test_axis.py

```diff
@@ -1962,91 +1741,6 @@ def axis360provider(
     return Axis360ProviderFixture(db, axis_files_fixture)
 
 
-class TestAxis360BibliographicCoverageProvider:
-    """Test the code that looks up bibliographic information from Axis 360."""
-
-    def test_script_instantiation(self, axis360provider: Axis360ProviderFixture):
-        """Test that RunCoverageProviderScript can instantiate
-        the coverage provider.
-        """
-        script = RunCollectionCoverageProviderScript(
-            Axis360BibliographicCoverageProvider,
-            axis360provider.db.session,
-            api_class=MockAxis360API,
-        )
-        [provider] = script.providers
-        assert isinstance(provider, Axis360BibliographicCoverageProvider)
-        assert isinstance(provider.api, MockAxis360API)
-
-    def test_process_item_creates_presentation_ready_work(
-        self, axis360provider: Axis360ProviderFixture
```

Comment:
I think it would be nice to preserve a test like this, that tests the whole process, making sure that the imported item actually lands in the DB.

Reply:
I brought this one back in.

Reply:
As well as the other one.